src/py/covidr/cli/db.py

```diff
@@ -0,0 +1,19 @@
+import click
+from covidr.database.connection import get_db_uri, init_db
+from covidr.database.schema import create_tables_and_schema
+from covidr.sandbox.runtimeenvironment import RuntimeEnvironment
+
+from .toplevel import cli
+
+
+@cli.group()
```

Comment:
interesting, going with click instead of argparse?

Reply:
I go back and forth. :)